src/databricks/labs/remorph/reconcile/exception.py

```diff
@@ -1,2 +1,6 @@
 class ColumnMismatchException(Exception):
     """Raise the error when there is a mismatch in source and target column names"""
+
+
+class MockDataNotAvailableException(Exception):
```

Comment:
The Exception is not for mock data but rather catching the real time exception into a clean error handling so you can mock the same.

check these examples: https://github.com/databrickslabs/lsql/blob/9032c9ddffd6b260265a733ae53d65c0c0157bc4/src/databricks/labs/lsql/backends.py#L276 
 we don't need to be this detailed, but need to cover them as common error buckets. I'm ok with `RuntimeError`

Reply:
done.